.github/workflows/connect-dev-release.yml

```diff
@@ -0,0 +1,57 @@
+name: "[Build/Test] connect-explorer"
```

Comment:
Will you be running the tests in this action as well? Is that the reason for build/test name?

Reply:
I guess it makes sense to have the tests in the same action that is building it so every-time we build it the test run.